build-npm-module/index180.js

```diff
@@ -8,7 +8,9 @@ const getSentenceCase = (str) => {
   return lowerStr.charAt(0).toUpperCase() + lowerStr.slice(1)
 };
 
-const properCaseArr = (str) => str.split(' ').map((word) => getSentenceCase(word));
-const getProperCase = (str) => properCaseArr.join(' ');
+const getProperCase = (str) => {
+  const properCaseArr = (str) => str.split(' ').map((word) => getSentenceCase(word));
+};
+
+// Call join on `properCaseArr` to convert the array to a string and return the result.
```

Comment:
Call the `join()` method on `properCaseArr` to convert the array into a string and return the result.